### PR TITLE
Bump version to 0.5.3

### PR DESCRIPTION
**Pull request by llamerada-jp**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ It should look like:
 (snip)
 ## [Unreleased]
 
+## [0.5.3]
```
Comment (ymmt2005) [inline RELEASE.md:21]: What's this?

File: docs/rancher.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ exit
 gcloud compute ssh --zone ${ZONE} worker2
 
 # Install lvmd
-TOPOLVM_VERSION=0.5.2
+TOPOLVM_VERSION=0.5.3
```
Comment (ymmt2005) [inline docs/rancher.md:168]: 0.5.2 remains in other places.

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,13 @@ This file itself is based on [Keep a CHANGELOG](https://keepachangelog.com/en/0.
 
 ## [Unreleased]
 
+## [0.5.3] - 2020-08-12
+
+### Changed
+
+- Fix to accept implicit StorageClassName (#182)
+- Fix documents (#172, #173, #174, #175, #176, #179)
+
```
Comment (binoue) [inline CHANGELOG.md:16]: Please add contributors section like below

https://github.com/topolvm/topolvm/blob/b0831898a2217e0b4e654f8a1449957def4d1d0f/CHANGELOG.md#fixed-2

File: RELEASE.md (hunk 1 above)
Comment (llamerada-jp) [reply RELEASE.md:21]: Before the change, It seems like the description was added to `Unreleased`.

Comment (ymmt2005) [reply RELEASE.md:21]: I can't understand.  Could you explain in other words?

Comment (llamerada-jp) [reply RELEASE.md]: Revert it, because there was no need to change.

**Pull request merged**